services/web/server/src/simcore_service_webserver/exporter/formatters/sds/text_files.py

```diff
@@ -0,0 +1,63 @@
+from typing import Union
+from pathlib import Path
+
+from pydantic import BaseModel, Field
+
+
+class BaseTextFile(BaseModel):
+    path: Union[str, Path] = Field(
```

Comment:
when you do that, you will always end up with a str. so I'm not sure I understand the reason of that Union...

better use a validator. you would spare the conversion to Path in all your functions.

Reply:
I later do a "base_path / path" operation. The "/" operator accepts both `str` and `Path`. I had some testing done before. doing this